employees/tests/test_views.py

```diff
@@ -140,7 +141,7 @@ def setUp(self):
         self.project = ProjectFactory()
         self.project.members.add(self.user)
         self.client.force_login(self.user)
-        self.report = ReportFactory(author=self.user, project=self.project)
+        self.report = ReportFactory(author=self.user, project=self.project, task_activities=self.task_type)
```

Comment:
`task_activities` should be now auto created in `ReportFactory`, please remove it as well as creating it at the beginning of `setUp` if it is not needed for particular reason.

Reply:
Done